process/cla_faq.md

```diff
@@ -0,0 +1,21 @@
+# Contributor License Agreement (CLA) FAQ
+
+Many of the projects hosted by ASWF leverage a Contributor License Agreement for contributors to the given project. These projects use [EasyCLA](https://lfcla.com/) for automating the process of signing and managing CLAs for the project. 
+
+This document captures some commonly asked questions by organizations on CLAs and using the EasyCLA system.
+
+Q: Why does the project require me to sign a CLA?
+
+A: It depends from project to project, but generally the project is looking to ensure that each indivudual is properly authorized to contribute to the project ( most specifically by thier employer ) and that any additional terms that a contributor should be bound to.
+
+Q: What is the difference between a Corporate and Individual CLA?
+
+A: The Corporate CLA is designed for an organization to authorize contributors that have permission to contribute on behalf of the organization. The Individual CLA is specific for an indivudual contributor, including those not affliated with an organization.
+
+Q: Who needs to sign the Corporate CLA?
+
+A: The Corporate CLA should be signed by someone with signatory rights in your organization. This might be an executive or legal council, but you should always consult your leadership to confirm.
```

Comment:
Suggest changing this to: "A: The Corporate CLA should be signed by someone in your organization who has sufficient authority to execute the CLA on its behalf. Each company or other organization has its own internal controls to determine who has this authority, so the project can't determine this for you; for example, it might be an executive or other member of your leadership, but you should always consult your organization's leadership to confirm."